Fabric.Authorization.Domain/Services/GroupService.cs

```diff
@@ -228,13 +243,21 @@ public async Task<Group> AddChildGroups(Group group, IEnumerable<string> childGr
                 throw new AlreadyExistsException<Group>($"The following groups are already children of group {group.Name}: {string.Join(", ", existingAssociations.Select(g => g.Name))}");
             }
 
-            return await _groupStore.AddChildGroups(group, childGroups);
+            var persistedGroup = await _groupStore.AddChildGroups(group, childGroups);
+            await this._roleManager.RefreshDosAdminRolesAsync(group.Children.SelectMany(childGroup => childGroup.Users));
+
+            return persistedGroup;
         }
 
         public async Task<Group> RemoveChildGroups(Group group, IEnumerable<string> childGroupNames)
         {
             var childGroups = await _groupStore.Get(childGroupNames);
-            return await _groupStore.RemoveChildGroups(group, childGroups);
+            var childUsers = childGroups.SelectMany(g => g.Users);
+
+            var persistedGroup = await _groupStore.RemoveChildGroups(group, childGroups);
+            await this._roleManager.RefreshDosAdminRolesAsync(childUsers);
```

Comment:
Similar to above,
1) ChildGroup navigation collection does not include Users, but this is irrelevant since AD groups cannot have users.
2) Remove `var childUsers = childGroups.SelectMany(g => g.Users);`
3) This call should then just be:
`await this._roleManager.RefreshDosAdminRolesAsync(group.Users);`